jcl/src/java.base/share/classes/java/lang/String.java

```diff
@@ -5282,5 +5283,15 @@ static void checkBoundsBeginEnd(int begin, int end, int length) {
 		}
 		throw newStringIndexOutOfBoundsException(begin, end, length);
 	}
+	
+	@Override
+	public IntStream chars() {
```

Comment:
Ideally there should be an implementation which is optimized for String, rather than using the generic CharSequence implementation. Same for codePoints() and the changes in StringBuffer and StringBuilder

Reply:
We will accept this default implementation to fix the signature test, with two things:
- add a comment to each method to mention the implementation needs to be updated
- open an Issue to cover updating the implementations

Reply:
Updated PR with comments and opened an issue https://github.com/eclipse/openj9/issues/642.